BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00330-01
```

Comment:
CodeFlow shows there is some extra whitespace (newline) in our version.  I recommend copying from CoreFx directly.  Not blocking.